2019-11-23_Angular/my-blog/src/app/component/validation-error/validation-error.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {AbstractControl} from '@angular/forms';
+
+@Component({
+  selector: 'app-validation-error',
+  templateUrl: './validation-error.component.html',
+  styleUrls: ['./validation-error.component.scss']
+})
+export class ValidationErrorComponent implements OnInit {
+
+  translations = [
```

Comment:
@x3matt Makes sense to place translations in some external file - later we can get those values from BE. And translations can be supplied as another parameter to the component.